### ISSUE TITLE
JS comments not highlighted under certain conditions

### ISSUE THREAD

**Issue by deerboy** [PrismJS/prism#1900]
**Information:**
- Prism version: latest
- Environment: Browser

**Description**
![image](https://user-images.githubusercontent.com/5842851/57977568-630b0500-7a36-11e9-9c36-8134dcab5432.png)

- If the string ends with `/ **`, the comments after that will not be highlighted
- JavaScript

**Example**

Please [paste](https://prismjs.com/test.html#language=javascript) the code below.

```js
const path = {
  scss: 'src/scss/**/*.scss',
  image: 'src/images/**', // By this line
};

// Comments are not highlighted
function css() {
  return src(path.scss)
    .pipe(dest('assets'));
}
```

**Comment by RunDevelopment** [comment]
This is caused by #1492.

**Comment by RunDevelopment** [comment]
Fixed by #2032.
